### ISSUE TITLE
Feature: left click tray = quick actions

### ISSUE THREAD

**Issue by patienttruth** [hass-agent/HASS.Agent#87]
Please make sure that the feature you'd like to request is not already requested :)

----

**Is your feature request related to a problem? Please describe.**
NA

**Describe the solution you'd like**
An option to enable quick actions to be displayed when you left or right click the tray icon.

**Describe alternatives you've considered**
NA

**Additional context**
Currently there are options to modify the left and right clicks of the tray. I don't believe there is any action defined for a single left click of the tray icon. Mine doesn't do anything at least.  A double click opens the main window. To get to quick actions I have to right click, and then select quick actions. I thought it would be nice to streamline the work flow.

Thanks for all the work!

**Comment by amadeo-alex** [comment]
Hmm, when WebView is enabled the left click can be configured to show the normal "right click menu":
![TtuO7e](https://github.com/hass-agent/HASS.Agent/assets/68441479/0120252a-eed3-4499-8a0b-c34bf9af6797)

I could make it that when WebView is not enabled the left click will open the quick actions window - I understand that the quick actions hothey is not exactly working for your use-case?

**Comment by patienttruth** [comment]
You're correct about the existing options. I just thought it was interesting that there was no default for a single left click and perhaps this was a good option. And yes, I suppose it would have to be in the case that webview wasn't enabled.

That's correct about hotkeys. To be honest I forgot about hotkeys, although I didn't realize that the panel could be opened with one. I believe I programmed one for a quick action. That said, quick access to the panel one way or another is what I was thinking would be nice.

**Comment by RangerCD** [comment]
Is it possible to switch left and right action? It would be helpful if I want to click something on HA while I'm using tablet, so I can show webview by touching tray icon instead of long pressing.

**Comment by RangerCD** [comment]
@gitrummy Cool! There should be an option for swapping buttons.

**Comment by Daniel-Vitanza** [comment]
I vote for this too ! Way easier to do a left-lick on the touchscreen of my windows tablet to check the dashbaoard with my security cams .
